src/main/java/duke/helptool/TaskList.java

```diff
@@ -0,0 +1,69 @@
+package duke.helptool;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+import duke.task.Task;
+
+/**
+ * The type Task list.
+ */
+public class TaskList {
+    private final ArrayList<Task> taskList;
+
+    /**
+     * Instantiates a new Task list.
+     *
+     * @param arrayList the array list
+     */
+    public TaskList(ArrayList<Task> arrayList) {
+        this.taskList = Objects.requireNonNullElseGet(arrayList, ArrayList::new);
```

Comment:
I like how you made use of a method within the Object class to check if the arrayList parameter is null or not

Reply:
Thanks :)